app/controllers/reminders_controller.rb

```diff
@@ -0,0 +1,29 @@
+class RemindersController < ApplicationController
+  before_action :set_reminder, only: [:update, :destroy]
+
+  def index
+    authorize Reminder
+    @reminders = Reminder.all
+  end
+
+  # Sends the email
+  def update
+    volunteer = @reminder.volunteer
+    email = @reminder.volunteer.contact.primary_email
+    ReminderMailer.reminder_email(volunteer, @reminder, email).deliver
```

Comment:
actually you don't need to pass email and volunteer to the reminder, as you can do all that within the mailer. The mailer can only take `@reminder`.
```ruby
ReminderMailer.reminder_email(@reminder).deliver
```